recipes/ipython-memory-usage/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ipython_memory_usage" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f4697210257f853fea74de0cf3fae60a32e550e578bac6f46de9b40c550422b
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - memory_profiler
+    - ipython
+
+test:
+  imports:
+    - ipython_memory_usage
+
+about:
+  home: http://github.com/simplejson/simplejson
+  license: BSD-2-Clause
+  license_family: BSD
+  license_file: {{ RECIPE_DIR }}/LICENSE
```

Comment:
Made an upstream issue to package the LICENSE: https://github.com/ianozsvald/ipython_memory_usage/issues/10